content/en/docs/measuring/metric-providers/carbon-intensity-electricitymaps-machine.md

```
@@ -0,0 +1,143 @@
+- `value`: The carbon intensity at that point in time in `gCO2e/kWh` as an integer.
+- `detail_name`: Always `electricity_maps` to indicate the data source.
+
+Any errors that occur while talking to the API are appended to the provider's stderr buffer and can
+be inspected in the run details in the frontend.
+
+### How it works
+
+On `start_profiling` and `stop_profiling` the provider records UTC timestamps. When the metrics are
+read at the end of the run it issues a single HTTP GET request to
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)